### DISCUSSION TITLE
Absolute numbers with kitty @ resize-window

### DISCUSSION THREAD

**Discussion by salmankhilji** [kovidgoyal/kitty#3624] [Ideas]
Would it be easy to support absolute numbers with the `@ resize-window` (and its cousin, `resize_window`) command?

Something like this:

```
kitty @ resize-window --dimension=12 --axis=vertical
kitty @ resize-window --absolute=12 --axis=vertical
```

**Comment by kovidgoyal** [comment]
No, layouts work via biases not absolute sizes.
